cli/src/cli_util.rs

```diff
@@ -2599,3 +2601,40 @@ impl CliRunner {
         exit_code
     }
 }
+
+fn map_clap_cli_error(
+    cmd_err: CommandError,
+    ui: &Ui,
+    layered_configs: &LayeredConfigs,
+) -> CommandError {
+    let CommandError::ClapCliError { err, hint: None } = &cmd_err else {
+        return cmd_err;
+    };
+    if let (Some(ContextValue::String(arg)), Some(ContextValue::String(value))) = (
+        err.get(ContextKind::InvalidArg),
+        err.get(ContextKind::InvalidValue),
+    ) {
+        if arg.as_str() == "--template <TEMPLATE>" && value.is_empty() {
+            // Suppress the error, it's less important than the original error.
+            if let Ok(template_aliases) = load_template_aliases(ui, layered_configs) {
+                return CommandError::ClapCliError {
+                    err: err.clone(),
+                    hint: Some(format_symbol_names_hint(&template_aliases)),
+                };
+            }
+        }
+    }
+    cmd_err
+}
+
+fn format_symbol_names_hint(template_aliases: &TemplateAliasesMap) -> String {
+    let mut hint = String::from("The following symbol names are defined:\n");
```

Comment:
nit: maybe better to say "template aliases" than "symbol names"?

"symbol names" is correct in parsing context, but user wouldn't think that way.